holidays/locale/en_US/LC_MESSAGES/US.po

```diff
@@ -0,0 +1,441 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# United States holidays.
```

Comment:
Maybe it would be better to use full name here - "United States of America" or even "United States of America (the)" as in ISO?

Reply:
Hmm, do we want to add `UnitedStatesOfAmerica` and slowly deprecates the current `UnitedStates` class as well ? mainly for consistency's sake (this was what we did for `Korea` -> `SouthKorea`)

Another case that I can think of is `GB`'s [UnitedKingdomOfGreatBritainAndNorthernIreland](https://www.iso.org/obp/ui/#iso:code:3166:GB) @KJhellico @arkid15r

Reply:
I don't think we should rename the class yet. Not all of our classes match the full names of the countries _exactly_. ;)